src/dotnet/CommandLine/CommandLineApplication.cs

```diff
@@ -64,7 +64,21 @@ public CommandLineApplication Command(string name, bool throwOnUnexpectedArg = t
         public CommandLineApplication Command(string name, Action<CommandLineApplication> configuration,
             bool throwOnUnexpectedArg = true)
         {
-            var command = new CommandLineApplication(throwOnUnexpectedArg) { Name = name, Parent = this };
+            var command = new CommandLineApplication(throwOnUnexpectedArg) { Name = name };
+            return Command(command, configuration, throwOnUnexpectedArg);
+        }
+
+        public CommandLineApplication Command(CommandLineApplication command, bool throwOnUnexpectedArg = true)
+        {
+            return Command(command, _ => { }, throwOnUnexpectedArg);
+        }
+
+        public CommandLineApplication Command(
+            CommandLineApplication command,
+            Action<CommandLineApplication> configuration,
+            bool throwOnUnexpectedArg = true)
```

Comment:
It's more of a code hygiene habit, not critical. Code doesn't need it until there's a bug upstream, at which time it becomes a pain to track down the issue.